### DISCUSSION TITLE
how can i use despine in seaborn 0.13

### DISCUSSION THREAD

**Discussion by z626093820** [mwaskom/seaborn#3526] [Q&A]
I want to use the Despine function In the seaborn 0.13 version, I want to remove the line of the upper right coordinate axis, how should I change,THANK YOU!
import seaborn.objects as so
import seaborn as sns
from seaborn import axes_style,plotting_context
from seaborn import despine
so.Plot.config.theme.update(axes_style("ticks")|plotting_context('paper'))
data = sns.load_dataset('penguins')
sns.despine()
((so.Plot(data, x="bill_length_mm", y="bill_depth_mm").layout(size=(3, 3))
 .add(so.Dot(), color="species").label(x="a", y="b",title="c")
 .add(so.Line(color="black"),so.PolyFit(), y="bill_depth_mm", label="depth"))
 .save(r'E:\下载\1.svg'))

![image](https://github.com/mwaskom/seaborn/assets/37371455/cc69a63a-1f86-4f00-b850-9392957eee17)

**Comment by mwaskom** [comment]
You can use the `axes.spines.{side}` keys in `Plot.theme` or `Plot.config.theme`:

```python
so.Plot.config.theme.update({**sns.axes_style("ticks"), "axes.spines.top": False, "axes.spines.right": False})
so.Plot()
```
![image](https://github.com/mwaskom/seaborn/assets/315810/548c1ba9-dcc8-493f-8002-2d5d4ebdb4a6)
